metrics.md

```diff
@@ -0,0 +1,85 @@
+# Universal Search Metrics
```

Comment:
This is a really good start, thanks Chuck.  Can you identify what the most important KPI is?  (probably something with engagement ratio?)